42.md

```diff
@@ -32,6 +32,8 @@ And, when sent by clients, the following form:
 ["AUTH", <signed-event-json>]
 ```
 
+Clients MAY provide signed events from all logged-in users in a sequence of `AUTH` messages. Relays MUST consider the client is blending the feeds of all users in one screen and reply accordingly.
```

Comment:
That second sentence is talking about a conscious relay? I'd word it more functional.

```suggestion
Clients MAY provide signed events from multiple pubkeys in a sequence of `AUTH` messages. Relays MUST treat all pubkeys as authenticated accordingly.
```